packages/space-opera/src/components/utils/create_object_url.ts

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
I assume these files are just copied over from the editing adapter?

Reply:
Yes indeed. I think I have most of what I need to keep now.